boards/arduino/nicla_vision/arduino_nicla_vision_stm32h747xx_m7.yaml

```diff
@@ -11,4 +11,5 @@ supported:
   - gpio
   - spi
   - i2c
+  - usbd
```

Comment:
Please move it to a separate commit "boards: nicla_vision: add usbd test feature"